widgets/multiple-choice-list/index.js

```diff
@@ -1,24 +1,61 @@
 var React = require('react');
+var cx = require('classnames');
 
 var utils = require('../../lib/widgetUtils.js');
-var defaultTemplate = require('./template.html');
 
-function multipleChoiceList({container = null, facetName = null, template = defaultTemplate}) {
+var defaultTemplate = `<label>
+  <input type="checkbox" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{name}} <span>{{count}}</span>
+</label>`;
+
+/**
+ * Instantiate a list of refinement based on a facet
+ * @param  {String|DOMElement} options.container Valid CSS Selector as a string or DOMElement
+ * @param  {String} options.facetName Name of the attribute for faceting
+ * @param  {String} options.operator How to apply refinements. Possible values: `or`, `and`
+ * @param  {String[]} [options.sortBy=['count:desc']] How to sort refinements. Possible values: `count|isRefined|name:asc|desc`
+ * @param  {String} [options.limit=100] How much facet values to get.
+ * @param  {String|String[]} [options.cssClass=null] CSS class(es) for the main `<ul>` wrapper.
+ * @param  {String|Function} [options.template] Item template, provided with `name`, `count`, `isRefined`
+ * @return {Object}
+ */
+function multipleChoiceList({
+    container = null,
+    facetName = null,
+    operator = null,
+    sortBy = ['count:desc'],
+    limit = 100,
+    cssClass = null,
+    template = defaultTemplate
+  }) {
+  var usage = 'Usage: multipleChoiceList({container, facetName, operator[sortBy, limit, cssClass, template]})';
+
   var MultipleChoiceList = require('../../components/MultipleChoiceList');
   var containerNode = utils.getContainerNode(container);
 
-  if (container === null || facetName === null) {
-    throw new Error('Usage: multipleChoiceList({container, facetName[, template]})');
+  if (container === null ||
+    facetName === null ||
+    operator === null) {
+    throw new Error(usage);
+  }
+
+  operator = operator.toLowerCase();
+  if (operator !== 'and' && operator !== 'or') {
+    throw new Error(usage);
   }
 
   return {
-    getConfiguration: () => ({disjunctiveFacets: [facetName]}),
+    getConfiguration: () => ({
+      maxValuesPerFacet: limit,
```

Comment:
This will fail because it's a global param amongst facets. For now I will have to .split() the results.